### DISCUSSION TITLE
Add migration guide to v1.0.0 release?

### DISCUSSION THREAD

**Discussion by colinking** [vercel/swr#1390] [General]
Would y'all add a link to the migration guide y'all released on your blog to the `1.0.0` GitHub release? https://swr.vercel.app/blog/swr-v1#migration-guide

I stumbled on it only after upgrading a project of mine and knowing this existed would have been helpful :) Thanks, and congrats on the milestone!

**Comment by huozhi** [comment]
Good idea! I've added the link into the [github release](https://github.com/vercel/swr/releases/tag/1.0.0).
Thanks for bringing this up 🙏

**Comment by shuding** [comment]
There's a link to the GitHub Release page here: https://swr.vercel.app/blog/swr-v1#changelog ;)
